data_steward/ci/test_datasets.py

```diff
@@ -18,41 +17,40 @@ def get_client(project_id, app_creds):
     """
     Ensure only one client is created and reused
 
-    :param project_id:  project to get a client for
-    :returns: a big query client object
+    :param project_id: project to get a client for
+    :param app_creds: Filepath to credentials file used to create the client
+    :returns: A BigQueryClient object
     """
     global CLIENT
     if not CLIENT:
         credentials = service_account.Credentials.from_service_account_file(
             app_creds)
-        CLIENT = bigquery.Client(project=project_id, credentials=credentials)
+        CLIENT = BigQueryClient(project_id=project_id, credentials=credentials)
 
     return CLIENT
```

Comment:
Nice work parametrizing client in every function! I hope we can retire this global client.

Reply:
It is possible to remove the `get_client(...)` function and just create the bq client inside `create_test_datasets(...)` and that still ensures that only one client is created and reused.